src/react-components/create-object-dialog.js

```diff
@@ -84,6 +78,14 @@ export default class CreateObjectDialog extends Component {
   };
 
   render() {
+    const { onCreate, onClose, ...other } = this.props;
+
+    const onCreateClicked = e => {
```

Comment:
Any reason you moved this away from being a method? In generail my rule is to only define functions in render if they need to capture some things in scope that would be ugly pass around (like itterators). Beyond that, if it uses `this` it's a method (which I bind n the contructior) otherwiset its a function I define outside the component.

Reply:
Honestly I think it was because otherwise lint complains that I didn't use `onCreate`. But I want to destructure `onCreate` so that `other` has the right props to forward onto `DialogContainer`.

How would you arrange this? Just make a lint ignore?